### ISSUE TITLE
Google Cloud SQL Connecting from Google Kubernetes Engine

### ISSUE THREAD

**Issue by brettcurtis** [pegasystems/pega-helm-charts#40]
Everything I've done to connect to my postgres instances to date have been using the Cloud SQL Proxy. https://cloud.google.com/sql/docs/postgres/connect-kubernetes-engine

I'm new to Helm just because I've found it hasn't been able to do everything I need so I just end up dealing with my own yaml or in most cases Terraform. However this is a pretty beefy app and I was hoping to use the helm charts. Any hope we can see support for cloud sql proxy? 

I download the templates here: https://github.com/pegasystems/pega-helm-charts/releases for GKE but again, a lot of work to get them going. Also I'm not sure if the terraform kubernetes provider is ready for everything in there yet, I need to look a bit closer.

**Comment by dcasavant** [comment]
Hi @brettcurtis ,

We haven't validate the Cloud SQL proxy yet, but it does seem like a good idea.  Do you have chart changes that incorporate it?  If not, I'll add it to the backlog.

**Comment by brettcurtis** [comment]
I don't - haven't done anything with helm until yesterday so I got a bit to learn. That being said Cloud SQL now supports private IPs so I'm not sure there is a need for the proxy anymore.  Looks to have some benefits https://cloud.google.com/sql/docs/mysql/private-ip#benefits. I'm content with this solution. 

I got it all running today but I'll have some followup questions that i will kick off in another thread. Thanks for quick response!! Feel free to close.
